packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,12 +10,10 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
```

Comment:
We can maybe move these under `1.46.0` still keeping the `Unreleased`

Reply:
Yeah, I was thinking to do it after we finish the testing but I'm ok to do it now too 😉 .